draft-ietf-jsonpath-base.md

```diff
@@ -1592,6 +1663,54 @@ Change controller:
 Provisional registration? (standards tree only):
 : no
 
+## Function Extensions {#iana-fnex}
+
+This specification defines a new "Function Extensions sub-registry" in
+a new "JSONPath Parameters registry", with the policy "expert review"
+({{Section 4.5 of -ianacons}}).
+
+The experts are instructed to be frugal in the allocation of very short
+function extension names, keeping them in reserve for applications that
```

Comment:
What do you mean by "applications" here?

Reply:
application in the sense of the way this is going to the used, usage

Reply:
I wonder if another word would work better here so that readers don't conflate it with executable applications or other consuming code.

Reply:
Perhaps:
```suggestion
function extension names, keeping them in reserve for functions that
```